_component_design/pills.md

```diff
@@ -0,0 +1,132 @@
+
+{% figure [caption:"The pill pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/pills/pills-hero.png){:width="1200px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+  
+</section> 
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default layout
+
+Pills are a component with a variety of use cases, so you can throw more or less anything you want into them to indicate something has been selected: keyword search, input text search, the name of a server you’ve selected. If it can be selected, you can make a pill of it. 
+
+A pill is different than a tag in a variety of ways, most important of which that you cannot remove a tag.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"The pill pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/pills/pills-default.png){:width="499px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+  
+</section> 
+
+## Specifications
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Style
+
+While it is conceivable that pills are larger or smaller than body font, by and large they will exist at body font. 
+
+Pills consist of a container, text, and an “x” icon to indicate removal. By default, they are `gray 400` background.
+
+### Spacing
+
+Pills have 16px of space on either side for breathing room, and 8px below and beside them when grouped together. They begin just 4px beneath other components to which they are associated – use the best top spacing for your particular use case. We encourage the use of left justification for pills.
+
+If there are a lot of pills bunched together – more than 15 – consider an alternative pattern or contact the Helix team.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"The pill pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/pills/pills-specs.png){:width="509px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+  
+</section> 
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### States
+
+Pills have 16px of space on either side for breathing room, and 8px below and beside them when grouped together. They begin just 4px beneath other components to which they are associated – use the best top spacing for your particular use case. We encourage the use of left justification for pills.
+
+If there are a lot of pills bunched together – more than 15 – consider an alternative pattern or contact the Helix team.
```

Comment:
Rewrite: "If more than 15 pills are bunched together, consider an alternative pattern or contact the Helix team."

Reply:
updated.